packaging_automation/prepare_release.py

```diff
@@ -0,0 +1,394 @@
+
+    project_version_details = get_version_details(project_version)
+    default_upcoming_version = get_upcoming_patch_version(project_version)
+    upcoming_version = os.getenv("UPCOMING_VERSION", default=default_upcoming_version)
+    upcoming_minor_version = get_minor_version(upcoming_version)
+    devel_version = f"{upcoming_minor_version}devel"
+
+    release_branch_name = f'release-{project_version_details["major"]}.{project_version_details["minor"]}'
+    release_branch_name = f"{release_branch_name}-test" if is_test else release_branch_name
+    upcoming_version_branch = f"master-update-version-{uuid.uuid4()}"
+
+    g = Github(github_token)
+    repository = g.get_repo(f"{REPO_OWNER}/{project_name}")
+    newly_created_sql_file = ""
+
+    # if major release
+    if is_major_release(project_version):
+        print(f"### {project_version} is a major release. Executing Major release flow###")
+        major_release_params = MajorReleaseParams(configure_in_path=configure_in_path, devel_version=devel_version,
+                                                  is_test=is_test, main_branch=main_branch,
+                                                  multi_extension_out_path=multi_extension_out_path,
+                                                  project_name=project_name, project_version=project_version,
+                                                  release_branch_name=release_branch_name)
+        prepare_release_branch_for_major_release(major_release_params)
+        branch_params = UpcomingVersionBranchParams(project_version=project_version,
+                                                    project_name=project_name,
+                                                    upcoming_version=upcoming_version,
+                                                    upcoming_version_branch=upcoming_version_branch,
+                                                    devel_version=devel_version, is_test=is_test,
+                                                    main_branch=main_branch,
+                                                    citus_control_file_path=citus_control_file_path,
+                                                    config_py_path=config_py_path,
+                                                    configure_in_path=configure_in_path,
+                                                    distributed_dir_path=distributed_dir_path,
+                                                    repository=repository,
+                                                    upcoming_minor_version=upcoming_minor_version,
+                                                    multi_extension_out_path=multi_extension_out_path,
+                                                    multi_extension_sql_path=multi_extension_sql_path)
+
+        newly_created_sql_file = prepare_upcoming_version_branch(branch_params)
+        print(f"OK {project_version} Major release flow executed successfully")
+    else:
```

Comment:
I do not see a comment here, so how is my comment above fixed exactly? See https://github.com/citusdata/tools/pull/91/files#r641466250

Reply:
I haven't pushed yet. First I write Fixed then I will push it in one commit. You are too fast for me :)